.github/dependabot.yml

```diff
@@ -0,0 +1,45 @@
+version: 2
+updates:
+  - package-ecosystem: gomod
+    directory: /
+    schedule:
+      interval: daily
+    groups:
+      k8s-libs:
+        patterns:
+          - "k8s.io/api"
```

Comment:
How about just concern k8s.io/kubernetes, or else there will be a lot of PRs.

Reply:
K8s.io/* package has undergone many version changes, so adding them to a group will update them as one PR.

Reply:
There is also a question that there maybe some conflicts when dependBot automatically submit PRs because the new version may change the interface or arguments, so maybe manually update version is better?

Reply:
Your concerns are reasonable. But most minor version updates are online compatible and will be merged only after passing all GitHub action automatic tests.

ref:
1. Group update pr case: https://github.com/prometheus-operator/prometheus-operator/pull/7405
2. Minor version updated case: https://github.com/prometheus-operator/prometheus-operator/pull/7401

Reply:
That's great！Minor version ok, what about major verion change?

Reply:
If the major version is backward compatible, it can also be merged automatically. 

If there is a conflict (maybe change the interface or arguments), you can reply to this PR `@dependabot ignore this major version` and handle it manually.

case: https://github.com/prometheus-operator/prometheus-operator/pull/7369#issuecomment-2713114432

Reply:
great!